### ISSUE TITLE
Feat(MUI): Use MUI Components

### ISSUE THREAD

**Issue by narainkarthikv** [narainkarthikv/readme-shop#8]
### 1. Update Layout & Navigation
- [x] Redesign the homepage for a cleaner, minimalist look.
- [x] Optimize navigation bar for better user flow.
- [x] Add drawer where applicable for better organization.
- [ ] Ensure responsive design across mobile and desktop.

### 2. Create Shopping Cart & Checkout
- [ ] Create shopping cart UI for clarity and ease of use.
- [ ] Enhance checkout flow with progress indicators and badge based counter.

**Comment by MonishKumar-max** [comment]
@narainkarthikv 
So, I have to use MUI and create the above listed components?

**Comment by narainkarthikv** [comment]
> [@narainkarthikv](https://github.com/narainkarthikv) So, I have to use MUI and create the above listed components?

Yes! for product related tasks - I've added like icons, badges, svgs - so make sure it is added in shopping cart.

**Comment by MonishKumar-max** [comment]
![Image](https://github.com/user-attachments/assets/05f911d7-8fbe-4490-8cb7-889065cec721)

![Image](https://github.com/user-attachments/assets/d445bdbb-7fd0-496a-b87c-3bf78eab288e)

font-family: 'Clash Grotesk', sans-serif;

**Comment by MonishKumar-max** [comment]
@narainkarthikv 
can I change this to your default fontfamily?

**Comment by MonishKumar-max** [comment]
@narainkarthikv 
I have completed working on the first point.
Please tell me what I should do in the second point?

**Comment by narainkarthikv** [comment]
> @narainkarthikv 
> I have completed working on the first point.
> Please tell me what I should do in the second point?
> Hey there! Add Badge counter on adding the components in shopping cart icon like each icons, badges, svg, trophies should count. Progress bar like couting each component - for now it is 5. So if badge is selected it should be 1/5, then icon selected 2/5. Similar to that.

**Comment by MonishKumar-max** [comment]
![Image](https://github.com/user-attachments/assets/45dad36d-1793-423d-b984-9d3807b4e60f)

@narainkarthikv 
But we can select as many icons or badges etc. here. Then why should I restrict it to 5?

**Comment by MonishKumar-max** [comment]
@narainkarthikv 

![Image](https://github.com/user-attachments/assets/0d08aab5-8558-4158-aa11-2de480734981)

Should I add it on top of this icon?

**Comment by MonishKumar-max** [comment]
@narainkarthikv 
Hey, I had completed the issue. Shall I submit a PR?
